src/app/modules/item/pages/item-details/item-details.component.html

```diff
@@ -48,16 +48,31 @@
       </a>
     </nav>
     <div class="bg-white">
-      <ng-container *ngIf="taskOptions$ | async as taskOptions">
-        <alg-item-content
-          *ngIf="contentTab.isActive || taskTabs.length > 0"
-          [hidden]="!contentTab.isActive"
-          [itemData]="itemData"
-          [taskOptions]="taskOptions"
-          [(taskView)]="taskView"
-          (taskTabsChange)="setTaskTabs($event)"
-          (scoreChange)="patchStateWithScore($event)"
-        ></alg-item-content>
+      <alg-error
+        *ngIf="unknownError"
+        i18n-message message="An unknown error occurred. If this problem persists, please contact us."
+      ></alg-error>
+
+      <ng-container *ngIf="contentTab.isActive || taskTabs.length > 0">
+        <alg-error *ngIf="answerFallbackLink$ | async as fallbackLink">
+          <ng-container *ngIf="(taskReadOnly$ | async); else fallback" i18n>Unable to load the answer</ng-container>
```

Comment:
This line is hard to test, it's very unlikely to happen in "real" life.
It can be tested locally by making `taskReadOnly$` an `Observable<true>` (in item-details.component.ts) and then loading a url with a bad answer id, ie: [localhost link](http://localhost:4200/#/activities/by-id/6379723280369399253;path=4702,1625159049301502151,5207993233955027100;parentAttempId=0;answerId=1/details)